storage/Volume.cc

```diff
@@ -2441,7 +2441,11 @@ namespace storage
     string Volume::getDmcryptName() const
     {
 	string nm;
-	if( !mp.empty() )
+        if ( mp == "/" )
```

Comment:
It was not about the space after the `if`, but about one extra space BEFORE the `if`. The whole new block looks to be more to the right than the rest of the code. But if indentation is already broken, then fine (for some definition of "fine"). :-)